File: openapi/code_samples/java/api_company_bankrupt_stats.java
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestParam;
import org.springframework.web.bind.annotation.RestController;

@RestController
public class CompanyController {

    @GetMapping("/api/company/top-10-companies-by-caen")
    public ResponseEntity<?> findTop10CompaniesByCaen(
            @RequestParam(name = "caen", defaultValue = "") String caen,
            @RequestParam(name = "year", defaultValue = "0") int year,
            @RequestParam(name = "county", defaultValue = "") String county,
            @RequestParam(name = "pageNumber", defaultValue = "0") int pageNumber,
            @RequestParam(name = "pageSize", defaultValue = "20") int pageSize,
            @RequestParam(name = "sortBy", defaultValue = "caen") String sortBy) {

        // Your implementation here

        return ResponseEntity.ok().build();
    }
}

File: openapi/code_samples/javascript/api_company_bankrupt_stats.js
const express = require('express');
const app = express();
const port = 3000;

app.get('/api/company/top-10-companies-by-caen', (req, res) => {
    const caen = req.query.caen || '';
    const year = parseInt(req.query.year) || 0;
    const county = req.query.county || '';
    const pageNumber = parseInt(req.query.pageNumber) || 0;
    const pageSize = parseInt(req.query.pageSize) || 20;
    const sortBy = req.query.sortBy || 'caen';

    // Your implementation here

    res.status(200).json({});
});

app.listen(port, () => {
    console.log(`Server is listening at http://localhost:${port}`);
});
